service/haproxy/run

```diff
@@ -38,7 +38,11 @@ reload() {
   socat /var/run/haproxy/socket - <<< "show servers state" > /var/state/haproxy/global
 
   # Trigger reload
-  haproxy -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $(pidof haproxy)
+  HAPROXY_PIDS=$(pidof haproxy)
+  haproxy -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $HAPROXY_PIDS
+  if [ -n "${HAPROXY_RELOAD_SIGTERM_DELAY-}" ]; then
+    sleep $HAPROXY_RELOAD_SIGTERM_DELAY && kill $HAPROXY_PIDS 2> /dev/null &
```

Comment:
`&` and `&&` are both posix shell pipeline separators. `&&` executes sequentially until a nonzero status. `&` executes in parallel. If the purpose of the code is to wait `$HAPROXY_RELOAD_SIGTERM_DELAY` seconds before killing `$HAPROXY_PIDS` that's what it'll do, if it's parent `run` is still executing after the delay. If `run` was for whatever reason killed/restarted in the meantime the kill won't be executed.